css/ryht-charter-campuses.css

```diff
@@ -534,6 +535,25 @@ p.moreinfo  {
     display: none;
 }
 
+input.autocompleteTextbox, select.dropdown {
+	display: block;
+	margin-top: 1ex;
+	margin-bottom: 1ex;
+}
+
+div.sidenav.tallpage p.moreinfo {
+	margin-top: 2em;
+	font-size: 14px;
+}
+
+div.sidenav.shortpage p.moreinfo {
+	margin-top: 0.5ex;
+	margin-bottom: 1ex;
+	padding-right: 8px;
+	font-size: 10px;
+	line-height: 11px;
+	overflow-y: auto;
+}
```

Comment:
The `shortpage` and `tallpage` classes get dynamically added and removed by the JS, based on viewport height.